odilia/src/commands.rs

```diff
@@ -7,43 +7,3 @@ type Request = Command;
 type Response = ();
```

Comment:
shouldn't this be something like Box<dyn IntoCommands>?

Reply:
No, the Resposne type for command-style handlers should be nothing on an error. There is nothing after the commands, they are the final point for any incoming events of any type.